spr/src/commands/diff.rs

```diff
@@ -446,7 +446,7 @@ async fn diff_impl(
         let new_base_branch_commit = jj.create_derived_commit(
             local_commit.parent_oid,
             &format!(
-                "[spr] {}\n\nCreated using jj-spr {}\n\n[skip ci]",
+                "[spr] {}\n\n[skip ci]",
```

Comment:
I feel like it would be nice if we could store this somewhere in the git metadata at least?

Reply:
Hi Lucio. Can you please explain why you want it? I am curious.

Reply:
To be able to track down what version of the tool potentially created some sort of bug